Lesson9/src/TextFormater.java

```diff
@@ -0,0 +1,72 @@
+import java.io.*;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class TextFormater implements Reader {
+    String str;
+    String[] words;
+    int count1 = 0;
+    int count2 = 0;
+    int count3 = 0;
+
+    @Override
+    public void readFile() {
+        try {
+            String filename;
+            Scanner scanner = new Scanner(System.in);
+            System.out.println("Введите название файла (text.txt):");
+            filename = scanner.nextLine();
+            FileReader fileReader = new FileReader("D://tms2021Android1//Lesson9//" + filename);
+            char[] buf = new char[256];
+            int i;
+            while ((i = fileReader.read(buf)) > 0) {
+                if (i < 256) {
+                    buf = Arrays.copyOf(buf, i);
+                }
+                str = new String(buf);
```

Comment:
you need to refactor this a bit. Use string builder as in example below

StringBuilder result = new StringBuilder();
            char[] buffer = new char[256];
            while ((i = fileReader.read(buffer)) > -1) {
                if (i < 256) {
                    buffer = Arrays.copyOf(buffer, i);
                }
                result.append(buffer);
            }
            return result.toString().getBytes();